packages/dht/src/connection/ConnectionManager.ts

```diff
@@ -127,6 +129,7 @@ export class ConnectionManager extends EventEmitter<TransportEvents> implements
     private rpcCommunicator?: RoutingRpcCommunicator
     private disconnectorIntervalRef?: NodeJS.Timeout
     private state = ConnectionManagerState.IDLE
+    private privateMode = false
```

Comment:
Maybe this could be renamed to "privateClientMode" to clarify that this is client-side setting? Also all the related methods should be renamed. A comment could be added to explain what this mode actually means.